### DISCUSSION TITLE
How to add Digital Signature (DSC) details of the Client

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#614] [QA] [answered]
Does Jamku provide DSC module where I can track and manage all the DSCs?
What are the features of this module?

**Comment by Divwork92** [comment, marked as answer]
There are 2 ways of adding DSC - 

(A) Via client dashboard
1. Go to client dashboard by clicking on the client name 
2. scroll down to folder on right side and click on '+'
3. select folder type as 'digital signature', add number (any number, should be unique), expiry date, folder name, location)
![image](https://user-images.githubusercontent.com/87230676/126492448-30d8bd28-3865-4238-9695-b52403dc1316.png)
4. click on save

(B) From folder list
1. Go to folder list
2. click on '+ADD'
3. select folder type as 'digital signature', add number (any number, should be unique), expiry date, folder name, location)
![image](https://user-images.githubusercontent.com/87230676/126492805-ce6bf51c-112a-4e6a-8741-ebb613a2cc45.png)
4. click on save

**You can manage from folder list, the ones in red are the expired 

You can track the location of the DSC, by clicking on the pen icon in front of the DCS and click on 'CHANGES'
![image](https://user-images.githubusercontent.com/87230676/126493772-3cda4180-c20a-4738-9ec3-dbfb5cffdb13.png)
